web/ui/react-app/src/components/withStartingIndicator.tsx

```diff
@@ -23,7 +23,7 @@ export const StartingContent: FC<StartingContentProps> = ({ status, isUnexpected
     <div className="text-center m-3">
       <div className="m-4">
         <h2>Starting up...</h2>
-        {status?.current! > status?.min! ? (
+        {status?.max! > 0 ? (
```

Comment:
I know I’ve worded this change as “the max value is not 0”, but I think I’d be good to check if it is greater than 0, which yields the same affect unless something weird happens with the server.